pkg/server/plugin/upstreamauthority/certmanager/certmanager.go

```diff
@@ -96,20 +127,16 @@ func (p *Plugin) SetLogger(log hclog.Logger) {
 }
 
 func (p *Plugin) Configure(_ context.Context, req *configv1.ConfigureRequest) (*configv1.ConfigureResponse, error) {
-	config, err := p.loadConfig(req)
+	newConfig, _, err := pluginconf.Build(req, p.buildConfig)
 	if err != nil {
 		return nil, err
 	}
 
-	if req.CoreConfiguration == nil {
-		return nil, status.Error(codes.InvalidArgument, "core configuration is required")
-	}
-
 	if req.CoreConfiguration.TrustDomain == "" {
 		return nil, status.Error(codes.InvalidArgument, "trust_domain is required")
 	}
```

Comment:
This is also checked by pluginconf.Build, correct?

Reply:
Yes, and I'm cutting in in the next push.

It was a remnant of the pre-`pluginconf.Build` approach.  Certmanager.go was one of the first plugins to be worked, and the `pluginconf.Build` came afterwards.